### PR TITLE
fix: cleanup files made by the tmp library in cypress.run()

### PR DESCRIPTION
**Pull request by alexsch01**
Without this PR, tmp files (in format "tmp-dynamicNum-dynamicString") made by the cypress.run() function will exist in the OS temp directory forever. I am using setGracefulCleanup by the tmp library

I have an example project on Linux here to showcase the default behavior: https://github.com/alexsch01/exampleForIssue

Let me know if you have any concerns

- Closes 

### PR Tasks

- [ ] Have tests been added/updated?
- [ ] Has the original issue (or this PR, if no issue exists) been tagged with a release in ZenHub? (user-facing changes only)
- [ ] Has a PR for user-facing changes been opened in [`cypress-documentation`](https://github.com/cypress-io/cypress-documentation)? 
- [ ] Have API changes been updated in the [`type definitions`](https://github.com/cypress-io/cypress/blob/develop/cli/types/cypress.d.ts)?

### CODE REVIEW

**Comment by alexsch01** [comment]
However, it would be nice if we didn't need to use temp files for this purpose.

**Comment by alexsch01** [comment]
Where in the cypress code is generating this (relative to the cypress.run() function)?
```
{"status":"finished","startedTestsAt":"2022-11-23T20:45:46.192Z","endedTestsAt":"2022-11-23T20:45:46.362Z","totalDuration":170,"totalSuites":0,"totalTests":1,"totalPassed":1,"totalPending":0,"totalFailed":0,"totalSkipped":0,"runs":[{"stats":{"suites":0,"tests":1,"passes":1,"pending":0,"skipped":0,"failures":0,"duration":170,"startedAt":"2022-11-23T20:45:46.192Z","endedAt":"2022-11-23T20:45:46.362Z"},"reporter":"cypress-mochawesome-reporter","reporterStats":{"suites":0,"tests":1,"passes":1,"pending":0,"failures":0,"start":"2022-11-23T20:45:46.197Z","end":"2022-11-23T20:45:46.375Z","duration":178,"testsRegistered":1,"passPercent":100,"pendingPercent":0,"other":0,"hasOther":false,"skipped":0,"hasSkipped":false},"hooks":[],"tests":[{"title":[" "],"state":"passed","body":"() => {\n  _console.default.log([1, 2], {\n    'hello1': 'world',\n    'hello2': 'world',\n    'hello3': 'world'\n  });\n}","displayError":null,"attempts":[{"state":"passed","error":null,"videoTimestamp":null,"duration":137,"startedAt":"2022-11-23T20:45:46.219Z","screenshots":[]}]}],"error":null,"video":null,"spec":{"fileExtension":".js","baseName":"spec.cy.js","fileName":"spec","specFileExtension":".cy.js","relativeToCommonRoot":"spec.cy.js","specType":"integration","name":"cypress/e2e/spec.cy.js","relative":"cypress\\e2e\\spec.cy.js","absolute":"C:/cypress-project/cypress/e2e/spec.cy.js"},"shouldUploadVideo":true}],"browserPath":"","browserName":"electron","browserVersion":"106.0.5249.51","osName":"win32","osVersion":"10.0.19045","cypressVersion":"11.0.1","config":{"reporter":"cypress-mochawesome-reporter","reporterOptions":{"reportDir":"cypress/reports","charts":true,"reportPageTitle":"Cypress Report","embeddedScreenshots":true,"inlineAssets":true,"quiet":true},"video":false,"viewportWidth":1920,"viewportHeight":1080,"setupNodeEvents":"[Function setupNodeEvents]","projectRoot":"C:\\cypress-project","projectName":"cypress-project","repoRoot":"C:/cypress-project","rawJson":{"e2e":{"setupNodeEvents":"[Function setupNodeEvents]"},"reporter":"cypress-mochawesome-reporter","reporterOptions":{"reportDir":"cypress/reports","charts":true,"reportPageTitle":"Cypress Report","embeddedScreenshots":true,"inlineAssets":true,"quiet":true},"video":false,"viewportWidth":1920,"viewportHeight":1080,"setupNodeEvents":"[Function setupNodeEvents]","envFile":{},"projectRoot":"C:\\cypress-project","projectName":"cypress-project","repoRoot":"C:/cypress-project"},"configFile":"cypress.config.js","morgan":false,"isTextTerminal":true,"socketId":"twsz6wye4v","report":true,"animationDistanceThreshold":5,"arch":"x64","baseUrl":null,"blockHosts":null,"chromeWebSecurity":true,"clientCertificates":[],"defaultCommandTimeout":4000,"downloadsFolder":"C:\\cypress-project\\cypress\\downloads","env":{"NTLM_AUTH_API":"http://127.0.0.1:60546","NTLM_AUTH_PROXY":"http://127.0.0.1:60547"},"execTimeout":60000,"experimentalFetchPolyfill":false,"experimentalInteractiveRunEvents":false,"experimentalSessionAndOrigin":false,"experimentalModifyObstructiveThirdPartyCode":false,"experimentalSourceRewriting":false,"experimentalSingleTabRunMode":false,"experimentalStudio":false,"experimentalWebKitSupport":false,"fileServerFolder":"C:\\cypress-project","fixturesFolder":"C:\\cypress-project\\cypress\\fixtures","excludeSpecPattern":"*.hot-update.js","includeShadowDom":false,"keystrokeDelay":0,"modifyObstructiveCode":true,"numTestsKeptInMemory":0,"platform":"win32","pageLoadTimeout":60000,"port":60550,"projectId":null,"redirectionLimit":20,"requestTimeout":5000,"resolvedNodePath":"C:\\Program Files\\nodejs\\node.exe","resolvedNodeVersion":"16.14.0","responseTimeout":30000,"retries":{"runMode":0,"openMode":0},"screenshotOnRunFailure":true,"screenshotsFolder":"C:\\cypress-project\\cypress\\screenshots","slowTestThreshold":10000,"scrollBehavior":"top","supportFile":"C:\\cypress-project\\cypress\\support\\e2e.js","supportFolder":"C:\\cypress-project\\cypress\\support","taskTimeout":60000,"testIsolation":null,"trashAssetsBeforeRuns":true,"userAgent":null,"videoCompression":32,"videosFolder":"C:\\cypress-project\\cypress\\videos","videoUploadOnPasses":true,"waitForAnimations":true,"watchForFileChanges":false,"specPattern":"cypress/e2e/**/*.cy.{js,jsx,ts,tsx}","additionalIgnorePattern":[],"autoOpen":false,"browsers":[{"name":"chrome","family":"chromium","channel":"stable","displayName":"Chrome","version":"107.0.5304.107","path":"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe","minSupportedVersion":64,"majorVersion":"107"},{"name":"edge","family":"chromium","channel":"stable","displayName":"Edge","version":"107.0.1418.56","path":"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe","minSupportedVersion":79,"majorVersion":"107"},{"name":"electron","channel":"stable","family":"chromium","displayName":"Electron","version":"106.0.5249.51","path":"","majorVersion":106}],"clientRoute":"/__/","cypressBinaryRoot":"C:\\Users\\USERNAME\\AppData\\Local\\Cypress\\Cache\\11.0.1\\Cypress\\resources\\app","devServerPublicPathRoute":"/__cypress/src","hosts":null,"isInteractive":true,"namespace":"__cypress","reporterRoute":"/__cypress/reporter","socketIoCookie":"__socket","socketIoRoute":"/__socket","version":"11.0.1","xhrRoute":"/xhrs/","cypressEnv":"production","resolved":{"animationDistanceThreshold":{"value":5,"from":"default"},"arch":{"value":"x64","from":"default"},"baseUrl":{"value":null,"from":"default"},"blockHosts":{"value":null,"from":"default"},"chromeWebSecurity":{"value":true,"from":"default"},"clientCertificates":{"value":[],"from":"default"},"defaultCommandTimeout":{"value":4000,"from":"default"},"downloadsFolder":{"value":"cypress/downloads","from":"default"},"env":{"NTLM_AUTH_API":{"value":"http://127.0.0.1:60546","from":"env"},"NTLM_AUTH_PROXY":{"value":"http://127.0.0.1:60547","from":"env"}},"execTimeout":{"value":60000,"from":"default"},"experimentalFetchPolyfill":{"value":false,"from":"default"},"experimentalInteractiveRunEvents":{"value":false,"from":"default"},"experimentalSessionAndOrigin":{"value":false,"from":"default"},"experimentalModifyObstructiveThirdPartyCode":{"value":false,"from":"default"},"experimentalSourceRewriting":{"value":false,"from":"default"},"experimentalSingleTabRunMode":{"value":false,"from":"default"},"experimentalStudio":{"value":false,"from":"default"},"experimentalWebKitSupport":{"value":false,"from":"default"},"fileServerFolder":{"value":"","from":"default"},"fixturesFolder":{"value":"cypress/fixtures","from":"default"},"excludeSpecPattern":{"value":"*.hot-update.js","from":"default"},"includeShadowDom":{"value":false,"from":"default"},"keystrokeDelay":{"value":0,"from":"default"},"modifyObstructiveCode":{"value":true,"from":"default"},"nodeVersion":{"from":"default"},"numTestsKeptInMemory":{"value":0,"from":"config"},"platform":{"value":"win32","from":"default"},"pageLoadTimeout":{"value":60000,"from":"default"},"port":{"value":null,"from":"default"},"projectId":{"value":null,"from":"default"},"redirectionLimit":{"value":20,"from":"default"},"reporter":{"value":"cypress-mochawesome-reporter","from":"config"},"reporterOptions":{"value":{"reportDir":"cypress/reports","charts":true,"reportPageTitle":"Cypress Report","embeddedScreenshots":true,"inlineAssets":true,"quiet":true},"from":"config"},"requestTimeout":{"value":5000,"from":"default"},"resolvedNodePath":{"value":null,"from":"default"},"resolvedNodeVersion":{"value":null,"from":"default"},"responseTimeout":{"value":30000,"from":"default"},"retries":{"value":{"runMode":0,"openMode":0},"from":"default"},"screenshotOnRunFailure":{"value":true,"from":"default"},"screenshotsFolder":{"value":"cypress/screenshots","from":"default"},"slowTestThreshold":{"value":10000,"from":"default"},"scrollBehavior":{"value":"top","from":"default"},"supportFile":{"value":"cypress/support/e2e.{js,jsx,ts,tsx}","from":"default"},"supportFolder":{"value":false,"from":"default"},"taskTimeout":{"value":60000,"from":"default"},"testIsolation":{"value":null,"from":"default"},"trashAssetsBeforeRuns":{"value":true,"from":"default"},"userAgent":{"value":null,"from":"default"},"video":{"value":false,"from":"config"},"videoCompression":{"value":32,"from":"default"},"videosFolder":{"value":"cypress/videos","from":"default"},"videoUploadOnPasses":{"value":true,"from":"default"},"viewportHeight":{"value":1080,"from":"config"},"viewportWidth":{"value":1920,"from":"config"},"waitForAnimations":{"value":true,"from":"default"},"watchForFileChanges":{"value":false,"from":"config"},"specPattern":{"value":"cypress/e2e/**/*.cy.{js,jsx,ts,tsx}","from":"default"},"browsers":{"value":[{"name":"chrome","family":"chromium","channel":"stable","displayName":"Chrome","version":"107.0.5304.107","path":"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe","minSupportedVersion":64,"majorVersion":"107"},{"name":"edge","family":"chromium","channel":"stable","displayName":"Edge","version":"107.0.1418.56","path":"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe","minSupportedVersion":79,"majorVersion":"107"},{"name":"electron","channel":"stable","family":"chromium","displayName":"Electron","version":"106.0.5249.51","path":"","majorVersion":106,"isHeadless":true,"isHeaded":false}],"from":"runtime"},"hosts":{"value":null,"from":"default"},"isInteractive":{"value":true,"from":"default"}},"testingType":"e2e","remote":{"origin":"http://localhost:60550","strategy":"file","fileServer":"http://localhost:60551","domainName":"localhost","props":null},"browser":null,"specs":[],"proxyUrl":"http://localhost:60550","browserUrl":"http://localhost:60550/__/","reporterUrl":"http://localhost:60550/__cypress/reporter","xhrUrl":"__cypress/xhrs/","proxyServer":"http://localhost:60550","state":{}}}
```

**Comment by alexsch01** [comment]
Where in the cypress code is generating this (relative to the cypress.run() function)?
```
{"status":"finished","startedTestsAt":"2022-12-04T14:41:00.409Z","endedTestsAt":"2022-12-04T14:41:00.410Z","totalDuration":1,"totalSuites":1,"totalTests":0,"totalPassed":0,"totalPending":0,"totalFailed":0,"totalSkipped":0,"runs":[{"stats":{"suites":1,"tests":0,"passes":0,"pending":0,"skipped":0,"failures":0,"duration":1,"startedAt":"2022-12-04T14:41:00.409Z","endedAt":"2022-12-04T14:41:00.410Z"},"reporter":"spec","reporterStats":{"suites":0,"tests":0,"passes":0,"pending":0,"failures":0,"start":"2022-12-04T14:41:00.410Z","end":"2022-12-04T14:41:00.412Z","duration":2},"hooks":[],"tests":[],"error":null,"video":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\videos\\spec.cy.js.mp4","spec":{"fileExtension":".js","baseName":"spec.cy.js","fileName":"spec","specFileExtension":".cy.js","relativeToCommonRoot":"spec.cy.js","specType":"integration","name":"cypress/e2e/spec.cy.js","relative":"cypress\\e2e\\spec.cy.js","absolute":"C:/Users/USERNAME/Downloads/exampleForIssue/cypress/e2e/spec.cy.js"},"shouldUploadVideo":true}],"browserPath":"","browserName":"electron","browserVersion":"106.0.5249.51","osName":"win32","osVersion":"10.0.19045","cypressVersion":"11.2.0","config":{"projectRoot":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue","projectName":"exampleForIssue","repoRoot":"C:/Users/USERNAME/Downloads/exampleForIssue","rawJson":{"e2e":{},"envFile":{},"projectRoot":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue","projectName":"exampleForIssue","repoRoot":"C:/Users/USERNAME/Downloads/exampleForIssue"},"configFile":"cypress.config.js","morgan":false,"isTextTerminal":true,"socketId":"524a57q08j","report":true,"animationDistanceThreshold":5,"arch":"x64","baseUrl":null,"blockHosts":null,"chromeWebSecurity":true,"clientCertificates":[],"defaultCommandTimeout":4000,"downloadsFolder":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\downloads","env":{},"execTimeout":60000,"experimentalFetchPolyfill":false,"experimentalInteractiveRunEvents":false,"experimentalRunAllSpecs":false,"experimentalSessionAndOrigin":false,"experimentalModifyObstructiveThirdPartyCode":false,"experimentalSourceRewriting":false,"experimentalSingleTabRunMode":false,"experimentalStudio":false,"experimentalWebKitSupport":false,"fileServerFolder":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue","fixturesFolder":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\fixtures","excludeSpecPattern":"*.hot-update.js","includeShadowDom":false,"keystrokeDelay":0,"modifyObstructiveCode":true,"numTestsKeptInMemory":0,"platform":"win32","pageLoadTimeout":60000,"port":61015,"projectId":null,"redirectionLimit":20,"reporter":"spec","reporterOptions":null,"requestTimeout":5000,"resolvedNodePath":"C:\\Program Files\\nodejs\\node.exe","resolvedNodeVersion":"16.14.0","responseTimeout":30000,"retries":{"runMode":0,"openMode":0},"screenshotOnRunFailure":true,"screenshotsFolder":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\screenshots","slowTestThreshold":10000,"scrollBehavior":"top","supportFile":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\support\\e2e.js","supportFolder":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\support","taskTimeout":60000,"testIsolation":null,"trashAssetsBeforeRuns":true,"userAgent":null,"video":true,"videoCompression":32,"videosFolder":"C:\\Users\\USERNAME\\Downloads\\exampleForIssue\\cypress\\videos","videoUploadOnPasses":true,"viewportHeight":660,"viewportWidth":1000,"waitForAnimations":true,"watchForFileChanges":false,"specPattern":"cypress/e2e/**/*.cy.{js,jsx,ts,tsx}","additionalIgnorePattern":[],"autoOpen":false,"browsers":[{"name":"chrome","family":"chromium","channel":"stable","displayName":"Chrome","version":"108.0.5359.72","path":"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe","minSupportedVersion":64,"majorVersion":"108"},{"name":"edge","family":"chromium","channel":"stable","displayName":"Edge","version":"107.0.1418.62","path":"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe","minSupportedVersion":79,"majorVersion":"107"},{"name":"electron","channel":"stable","family":"chromium","displayName":"Electron","version":"106.0.5249.51","path":"","majorVersion":106}],"clientRoute":"/__/","cypressBinaryRoot":"C:\\Users\\USERNAME\\AppData\\Local\\Cypress\\Cache\\11.2.0\\Cypress\\resources\\app","devServerPublicPathRoute":"/__cypress/src","hosts":null,"isInteractive":true,"namespace":"__cypress","reporterRoute":"/__cypress/reporter","socketIoCookie":"__socket","socketIoRoute":"/__socket","version":"11.2.0","xhrRoute":"/xhrs/","cypressEnv":"production","resolved":{"animationDistanceThreshold":{"value":5,"from":"default"},"arch":{"value":"x64","from":"default"},"baseUrl":{"value":null,"from":"default"},"blockHosts":{"value":null,"from":"default"},"chromeWebSecurity":{"value":true,"from":"default"},"clientCertificates":{"value":[],"from":"default"},"defaultCommandTimeout":{"value":4000,"from":"default"},"downloadsFolder":{"value":"cypress/downloads","from":"default"},"env":{},"execTimeout":{"value":60000,"from":"default"},"experimentalFetchPolyfill":{"value":false,"from":"default"},"experimentalInteractiveRunEvents":{"value":false,"from":"default"},"experimentalRunAllSpecs":{"value":false,"from":"default"},"experimentalSessionAndOrigin":{"value":false,"from":"default"},"experimentalModifyObstructiveThirdPartyCode":{"value":false,"from":"default"},"experimentalSourceRewriting":{"value":false,"from":"default"},"experimentalSingleTabRunMode":{"value":false,"from":"default"},"experimentalStudio":{"value":false,"from":"default"},"experimentalWebKitSupport":{"value":false,"from":"default"},"fileServerFolder":{"value":"","from":"default"},"fixturesFolder":{"value":"cypress/fixtures","from":"default"},"excludeSpecPattern":{"value":"*.hot-update.js","from":"default"},"includeShadowDom":{"value":false,"from":"default"},"keystrokeDelay":{"value":0,"from":"default"},"modifyObstructiveCode":{"value":true,"from":"default"},"nodeVersion":{"from":"default"},"numTestsKeptInMemory":{"value":0,"from":"config"},"platform":{"value":"win32","from":"default"},"pageLoadTimeout":{"value":60000,"from":"default"},"port":{"value":null,"from":"default"},"projectId":{"value":null,"from":"default"},"redirectionLimit":{"value":20,"from":"default"},"reporter":{"value":"spec","from":"default"},"reporterOptions":{"value":null,"from":"default"},"requestTimeout":{"value":5000,"from":"default"},"resolvedNodePath":{"value":null,"from":"default"},"resolvedNodeVersion":{"value":null,"from":"default"},"responseTimeout":{"value":30000,"from":"default"},"retries":{"value":{"runMode":0,"openMode":0},"from":"default"},"screenshotOnRunFailure":{"value":true,"from":"default"},"screenshotsFolder":{"value":"cypress/screenshots","from":"default"},"slowTestThreshold":{"value":10000,"from":"default"},"scrollBehavior":{"value":"top","from":"default"},"supportFile":{"value":"cypress/support/e2e.{js,jsx,ts,tsx}","from":"default"},"supportFolder":{"value":false,"from":"default"},"taskTimeout":{"value":60000,"from":"default"},"testIsolation":{"value":null,"from":"default"},"trashAssetsBeforeRuns":{"value":true,"from":"default"},"userAgent":{"value":null,"from":"default"},"video":{"value":true,"from":"default"},"videoCompression":{"value":32,"from":"default"},"videosFolder":{"value":"cypress/videos","from":"default"},"videoUploadOnPasses":{"value":true,"from":"default"},"viewportHeight":{"value":660,"from":"default"},"viewportWidth":{"value":1000,"from":"default"},"waitForAnimations":{"value":true,"from":"default"},"watchForFileChanges":{"value":false,"from":"config"},"specPattern":{"value":"cypress/e2e/**/*.cy.{js,jsx,ts,tsx}","from":"default"},"browsers":{"value":[{"name":"chrome","family":"chromium","channel":"stable","displayName":"Chrome","version":"108.0.5359.72","path":"C:\\Program Files (x86)\\Google\\Chrome\\Application\\chrome.exe","minSupportedVersion":64,"majorVersion":"108"},{"name":"edge","family":"chromium","channel":"stable","displayName":"Edge","version":"107.0.1418.62","path":"C:\\Program Files (x86)\\Microsoft\\Edge\\Application\\msedge.exe","minSupportedVersion":79,"majorVersion":"107"},{"name":"electron","channel":"stable","family":"chromium","displayName":"Electron","version":"106.0.5249.51","path":"","majorVersion":106,"isHeadless":true,"isHeaded":false}],"from":"runtime"},"hosts":{"value":null,"from":"default"},"isInteractive":{"value":true,"from":"default"}},"testingType":"e2e","remote":{"origin":"http://localhost:61015","strategy":"file","fileServer":"http://localhost:61016","domainName":"localhost","props":null},"browser":null,"specs":[],"proxyUrl":"http://localhost:61015","browserUrl":"http://localhost:61015/__/","reporterUrl":"http://localhost:61015/__cypress/reporter","xhrUrl":"__cypress/xhrs/","proxyServer":"http://localhost:61015","state":{}}}
```

**Comment by alexsch01** [comment]
Where in the cypress code is it generating this (relative to the cypress.run() function)?
```
{"status":"finished","startedTestsAt":"2022-12-04T16:25:58.105Z","endedTestsAt":"2022-12-04T16:25:58.105Z","totalDuration":0,"totalSuites":1,"totalTests":0,"totalPassed":0,"totalPending":0,"totalFailed":0,"totalSkipped":0,"runs":[{"stats":{"suites":1,"tests":0,"passes":0,"pending":0,"skipped":0,"failures":0,"duration":0,"startedAt":"2022-12-04T16:25:58.105Z","endedAt":"2022-12-04T16:25:58.105Z"},"reporter":"spec","reporterStats":{"suites":0,"tests":0,"passes":0,"pending":0,"failures":0,"start":"2022-12-04T16:25:58.106Z","end":"2022-12-04T16:25:58.107Z","duration":1},"hooks":[],"tests":[],"error":null,"video":null,"spec":{"fileExtension":".js","baseName":"spec.cy.js","fileName":"spec","specFileExtension":".cy.js","relativeToCommonRoot":"spec.cy.js","specType":"integration","name":"cypress/e2e/spec.cy.js","relative":"cypress/e2e/spec.cy.js","absolute":"/home/alex/exampleForIssue/cypress/e2e/spec.cy.js"},"shouldUploadVideo":true}],"browserPath":"","browserName":"electron","browserVersion":"106.0.5249.51","osName":"linux","osVersion":"Debian - ","cypressVersion":"11.2.0","config":{"video":false,"setupNodeEvents":"[Function setupNodeEvents]","projectRoot":"/home/alex/exampleForIssue","projectName":"exampleForIssue","repoRoot":"/home/alex/exampleForIssue","rawJson":{"e2e":{"setupNodeEvents":"[Function setupNodeEvents]"},"video":false,"setupNodeEvents":"[Function setupNodeEvents]","envFile":{},"projectRoot":"/home/alex/exampleForIssue","projectName":"exampleForIssue","repoRoot":"/home/alex/exampleForIssue"},"configFile":"cypress.config.js","morgan":false,"isTextTerminal":true,"socketId":"181wskwqc0","report":true,"animationDistanceThreshold":5,"arch":"x64","baseUrl":null,"blockHosts":null,"chromeWebSecurity":true,"clientCertificates":[],"defaultCommandTimeout":4000,"downloadsFolder":"/home/alex/exampleForIssue/cypress/downloads","env":{},"execTimeout":60000,"experimentalFetchPolyfill":false,"experimentalInteractiveRunEvents":false,"experimentalRunAllSpecs":false,"experimentalSessionAndOrigin":false,"experimentalModifyObstructiveThirdPartyCode":false,"experimentalSourceRewriting":false,"experimentalSingleTabRunMode":false,"experimentalStudio":false,"experimentalWebKitSupport":false,"fileServerFolder":"/home/alex/exampleForIssue","fixturesFolder":"/home/alex/exampleForIssue/cypress/fixtures","excludeSpecPattern":"*.hot-update.js","includeShadowDom":false,"keystrokeDelay":0,"modifyObstructiveCode":true,"numTestsKeptInMemory":0,"platform":"linux","pageLoadTimeout":60000,"port":37561,"projectId":null,"redirectionLimit":20,"reporter":"spec","reporterOptions":null,"requestTimeout":5000,"resolvedNodePath":"/usr/bin/node","resolvedNodeVersion":"18.12.1","responseTimeout":30000,"retries":{"runMode":0,"openMode":0},"screenshotOnRunFailure":true,"screenshotsFolder":"/home/alex/exampleForIssue/cypress/screenshots","slowTestThreshold":10000,"scrollBehavior":"top","supportFile":"/home/alex/exampleForIssue/cypress/support/e2e.js","supportFolder":"/home/alex/exampleForIssue/cypress/support","taskTimeout":60000,"testIsolation":null,"trashAssetsBeforeRuns":true,"userAgent":null,"videoCompression":32,"videosFolder":"/home/alex/exampleForIssue/cypress/videos","videoUploadOnPasses":true,"viewportHeight":660,"viewportWidth":1000,"waitForAnimations":true,"watchForFileChanges":false,"specPattern":"cypress/e2e/**/*.cy.{js,jsx,ts,tsx}","additionalIgnorePattern":[],"autoOpen":false,"browsers":[{"name":"electron","channel":"stable","family":"chromium","displayName":"Electron","version":"106.0.5249.51","path":"","majorVersion":106}],"clientRoute":"/__/","cypressBinaryRoot":"/home/alex/.cache/Cypress/11.2.0/Cypress/resources/app","devServerPublicPathRoute":"/__cypress/src","hosts":null,"isInteractive":true,"namespace":"__cypress","reporterRoute":"/__cypress/reporter","socketIoCookie":"__socket","socketIoRoute":"/__socket","version":"11.2.0","xhrRoute":"/xhrs/","cypressEnv":"production","resolved":{"animationDistanceThreshold":{"value":5,"from":"default"},"arch":{"value":"x64","from":"default"},"baseUrl":{"value":null,"from":"default"},"blockHosts":{"value":null,"from":"default"},"chromeWebSecurity":{"value":true,"from":"default"},"clientCertificates":{"value":[],"from":"default"},"defaultCommandTimeout":{"value":4000,"from":"default"},"downloadsFolder":{"value":"cypress/downloads","from":"default"},"env":{},"execTimeout":{"value":60000,"from":"default"},"experimentalFetchPolyfill":{"value":false,"from":"default"},"experimentalInteractiveRunEvents":{"value":false,"from":"default"},"experimentalRunAllSpecs":{"value":false,"from":"default"},"experimentalSessionAndOrigin":{"value":false,"from":"default"},"experimentalModifyObstructiveThirdPartyCode":{"value":false,"from":"default"},"experimentalSourceRewriting":{"value":false,"from":"default"},"experimentalSingleTabRunMode":{"value":false,"from":"default"},"experimentalStudio":{"value":false,"from":"default"},"experimentalWebKitSupport":{"value":false,"from":"default"},"fileServerFolder":{"value":"","from":"default"},"fixturesFolder":{"value":"cypress/fixtures","from":"default"},"excludeSpecPattern":{"value":"*.hot-update.js","from":"default"},"includeShadowDom":{"value":false,"from":"default"},"keystrokeDelay":{"value":0,"from":"default"},"modifyObstructiveCode":{"value":true,"from":"default"},"nodeVersion":{"from":"default"},"numTestsKeptInMemory":{"value":0,"from":"config"},"platform":{"value":"linux","from":"default"},"pageLoadTimeout":{"value":60000,"from":"default"},"port":{"value":null,"from":"default"},"projectId":{"value":null,"from":"default"},"redirectionLimit":{"value":20,"from":"default"},"reporter":{"value":"spec","from":"default"},"reporterOptions":{"value":null,"from":"default"},"requestTimeout":{"value":5000,"from":"default"},"resolvedNodePath":{"value":null,"from":"default"},"resolvedNodeVersion":{"value":null,"from":"default"},"responseTimeout":{"value":30000,"from":"default"},"retries":{"value":{"runMode":0,"openMode":0},"from":"default"},"screenshotOnRunFailure":{"value":true,"from":"default"},"screenshotsFolder":{"value":"cypress/screenshots","from":"default"},"slowTestThreshold":{"value":10000,"from":"default"},"scrollBehavior":{"value":"top","from":"default"},"supportFile":{"value":"cypress/support/e2e.{js,jsx,ts,tsx}","from":"default"},"supportFolder":{"value":false,"from":"default"},"taskTimeout":{"value":60000,"from":"default"},"testIsolation":{"value":null,"from":"default"},"trashAssetsBeforeRuns":{"value":true,"from":"default"},"userAgent":{"value":null,"from":"default"},"video":{"value":false,"from":"config"},"videoCompression":{"value":32,"from":"default"},"videosFolder":{"value":"cypress/videos","from":"default"},"videoUploadOnPasses":{"value":true,"from":"default"},"viewportHeight":{"value":660,"from":"default"},"viewportWidth":{"value":1000,"from":"default"},"waitForAnimations":{"value":true,"from":"default"},"watchForFileChanges":{"value":false,"from":"config"},"specPattern":{"value":"cypress/e2e/**/*.cy.{js,jsx,ts,tsx}","from":"default"},"browsers":{"value":[{"name":"electron","channel":"stable","family":"chromium","displayName":"Electron","version":"106.0.5249.51","path":"","majorVersion":106,"isHeadless":true,"isHeaded":false}],"from":"runtime"},"hosts":{"value":null,"from":"default"},"isInteractive":{"value":true,"from":"default"}},"testingType":"e2e","remote":{"origin":"http://localhost:37561","strategy":"file","fileServer":"http://localhost:45171","domainName":"localhost","props":null},"browser":null,"specs":[],"proxyUrl":"http://localhost:37561","browserUrl":"http://localhost:37561/__/","reporterUrl":"http://localhost:37561/__cypress/reporter","xhrUrl":"__cypress/xhrs/","proxyServer":"http://localhost:37561","state":{}}}
```
I used my repo (https://github.com/alexsch01/exampleForIssue) to generate this temp file

File: cli/lib/cypress.js [hunk 1]
```diff
@@ -31,6 +31,7 @@ const cypressModuleApi = {
 
     options = util.normalizeModuleOptions(options)
 
+    tmp.setGracefulCleanup()
```
Comment (mjhenkes) [inline cli/lib/cypress.js:34]: To fix linting:

```suggestion
    tmp.setGracefulCleanup()
    
```

**Comment by alexsch01** [comment]
@mjhenkes 

The CircleCI linux-lint test is failing
/root/cypress/cli/lib/cypress.js
35:1  error  Trailing spaces not allowed  no-trailing-spaces

The CircleCI driver-integration-tests-electron is failing
e2e/origin/navigation.cy.ts file has one failure

File: cli/lib/cypress.js [hunk 2]
```diff
@@ -31,6 +31,8 @@ const cypressModuleApi = {
...
+
```
Comment (mjhenkes) [inline cli/lib/cypress.js]:
```suggestion
    tmp.setGracefulCleanup()

```

**Comment by BlueWinds** [comment]
This looks like a good change to me, and verified the fix with the repro repo (thank you for providing!). Poking the rest of the team for a second review.

**Review by BlueWinds — APPROVED** [review]

**Review by mjhenkes — APPROVED** [review]

**Pull request merged**